### PR TITLE
Added card component and code snippet

### PR DESCRIPTION
**Pull request by poojavishnoi**

### CODE REVIEW

File: components/Card/card.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (sonishreyas) [inline components/Card/card.html:6]: meta author tag can be added here

File: doc/docstyle.css [hunk 2]
```diff
@@ -254,26 +254,24 @@ img {
...
+  .code_container {
     padding: 2rem;
-    
   }
-  
-  .code_container iframe{
-    width: 390px; height: 200px;
+
+  .code_container iframe {
+    width: 390px;
```
Comment (sonishreyas) [inline doc/docstyle.css:294]: you can make width as 100 % and height auto to make it responsive

**Review by sonishreyas — APPROVED** [review]
I had a couple of suggestions I have kept which can be done later. Rest it looks good to me

File: components/Card/card.html (hunk 1 above)
Comment (poojavishnoi) [reply components/Card/card.html:6]: Okay

File: doc/docstyle.css (hunk 2 above)
Comment (poojavishnoi) [reply doc/docstyle.css:294]: It is still responsive but I'll try this also.

File: doc/doc.html [hunk 3]
```diff
@@ -761,15 +760,218 @@ <h3>Shop with exclusive offers</h3>
 
      
     </div>
+<!--
```
Comment (AtulHande03a) [inline doc/doc.html]: remove if not required

File: doc/doc.html [hunk 4]
```diff
@@ -761,15 +760,218 @@ <h3>Shop with exclusive offers</h3>
...
+          style="border: 0; overflow: hidden"
```
Comment (AtulHande03a) [inline doc/doc.html]: avoid inline stylying , provide a class for this and use across the snippets

**Review by AtulHande03a — APPROVED** [review]

File: doc/doc.html (hunk 4 above)
Comment (poojavishnoi) [reply doc/doc.html]: Okay.

File: doc/doc.html (hunk 3 above)
Comment (poojavishnoi) [reply doc/doc.html]: done

**Pull request merged**